tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1239,9 +1239,9 @@ a dictionary can also be used with a named enum.
         # Use constant dictionary functions
```

Comment:
```suggestion
        # Use dictionary methods:
```

This refers to methods with the `const` qualifier, but they are not currently evaluated as constant expressions, even when called on constant dictionaries or named enums.